packages/core/src/internal/execution/future-processor/handlers/send-transaction.ts

```diff
@@ -120,10 +122,12 @@ export async function sendTransaction(
       transaction: result.transaction,
       nonce: result.nonce,
     };
+  } else {
+    throw new IgnitionError(ERRORS.EXECUTION.SIMULATION_ERROR, {
+      error:
+        typeof result.error === "string"
+          ? result.error
+          : failedEvmExecutionResultToErrorDescription(result.error),
+    });
   }
-
-  return createExecutionStateCompleteMessageForExecutionsWithOnchainInteractions(
```

Comment:
As I recall (with all that implies), I think we intentionally exclude simulation errors from the journal to allow for rerunning during debugging without increasing the size of the journal.

Is it possible we can trigger the equivalent event as would be triggered if we did record it to the journal?